nuxeo-drive-core/src/main/java/org/nuxeo/drive/listener/NuxeoDriveSyncRootCopyListener.java

```diff
@@ -54,6 +54,8 @@ public void handleEvent(Event event) {
         DocumentModelList syncRoots = getSyncRoots(doc, session);
         for (DocumentModel syncRoot : syncRoots) {
             syncRoot.setPropertyValue(NuxeoDriveManagerImpl.DRIVE_SUBSCRIPTIONS_PROPERTY, null);
+            syncRoot.putContextData("source", "drive");
+            syncRoot.putContextData(CoreSession.SOURCE, "drive");
```

Comment:
Indeed, I removed the first one. It's needed if we want to have former drive versioning rules in new system. If you have a look to [this](https://github.com/nuxeo/nuxeo-drive-server/pull/21/files#diff-2b666e0a45109ff340a90e52aedaaeb8R17), we use this `source` value for a versioning filter, used in a versioning policy.